stripe/res/values-zh/strings.xml

```diff
@@ -2,4 +2,5 @@
 <resources>
     <string name="acc_label_card_number">"卡号"</string>
     <string name="acc_label_expiry_date">"有效日期"</string>
+    <string name="expiry_date_hint">月 / 年</string>
```

Comment:
Not sure if I should have a space here for the Chinese version -- in JS, everything has a space, and ours never have. This was the only one where I kept it because I have no idea what those characters stand for, or if putting them near a slash was grammatically wrong or something